nist/int.go

```diff
@@ -140,7 +140,9 @@ func (i *Int) Set(a abstract.Scalar) abstract.Scalar {
 }
 
 func (i *Int) Clone() abstract.Scalar {
-	return &Int{V: i.V, M: i.M, BO: i.BO}
+	v := new(big.Int).Set(&i.V)
+	m := new(big.Int).Set(i.M)
+	return &Int{V: *v, M: m, BO: i.BO}
```

Comment:
yes, so we have to add that, too... I saw it does not correctly marshal/unmarshal 'my' way.